src/autocluster.erl

```diff
@@ -371,19 +382,34 @@ startup_delay(Max) ->
 
 -spec backend_register(#startup_state{}) -> ok | {error, iolist()}.
 backend_register(#startup_state{backend_module = Mod}) ->
-    Mod:register().
+    case erlang:function_exported(Mod, register, 0) of
```

Comment:
Why not simply check if `Mod` is `unconfigured` ? All backends should support these callbacks, otherwise is a proper error.